clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -746,6 +746,37 @@ public void testSendingLeaveGroupHeartbeatWhenPreviousOneInFlight(final short ve
         NetworkClientDelegate.PollResult pollAgain = heartbeatRequestManager.poll(time.milliseconds());
         assertEquals(0, pollAgain.unsentRequests.size());
     }
+    
+    @ParameterizedTest
+    @ApiKeyVersionsSource(apiKey = ApiKeys.CONSUMER_GROUP_HEARTBEAT)
+    public void testConsumerAcksReconciledAssignmentWithFirstHeartBeatAckLost(final short version) {
```

Comment:
nit: would this be simpler/clearer maybe? testConsumerAcksReconciledAssignmentAfterAckLost